buildlib/az-io_demo.sh

```diff
@@ -34,7 +34,7 @@ if [ "x$server_ip" = "x" ]; then
     if ! "${workspace}/../test/apps/iodemo/io_demo" |& add_timestamp >server.log & then
         error "Failed to start server"
     fi
-    server_pid=$(pgrep -u "$USER" -f iodemo)
+    server_pid=$(pgrep -u $USER -f 'io_demo$')
```

Comment:
same with corrupter?

Reply:
For corrupter it looks quite specific to me:
```sh
corrupter_pid=$(pgrep -u "$USER" -f 'network-corrupter')
```

And it doesn't spawn any new processes with similar names.

Reply:
maybe 
`corrupter_pid=$(pgrep -u "$USER" -f 'network-corrupter$')`
or
`corrupter_pid=$(pgrep -u "$USER" -f 'network-corrupter.sh$')`

Reply:
Done.

Reply:
@yosefe had to revert both changes, `$` doesn't work when running in Azure, I don't know why.